eclipse-scout-chart/src/chart/ChartJsRenderer.js

```diff
@@ -1008,6 +1008,10 @@ export default class ChartJsRenderer extends AbstractChartRenderer {
     if (isTooltipShowing) {
       this._renderTooltipLater(context);
     } else {
+      // clear timeout before creating a new handler.
+      // Otherwise, changing the context within the tooltip delay time creates a second handler
+      // and the first one will always be executed, since the tooltipTimoutId reference to it is lost
+      clearTimeout(this._tooltipTimeoutId);
```

Comment:
The tooltip will not be rendered until the user stops moving the mouse. This is not the intended behaviour.